src/Integration.Vsix.UnitTests/Analysis/IssueHandlerTests.cs

```diff
@@ -46,20 +39,18 @@ public void HandleNewIssues_UpdatedSnapshotAndHotspotStoreHaveExpectedValues()
         {
             var hotspotStoreMock = new Mock<ILocalHotspotsStoreUpdater>();
 
-            var hotspot = CreateIssue("S112", startLine: 1, endLine: 1, isHotspot: true);
```

Comment:
Please rename the test as well: HandleNewIssues_UpdatedSnapshotHasExpectedValues